src/components/SubHeader/SubHeader.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+
+import {
+  Header,
+  Title,
+  PrimaryNav,
+  NavDropDownButton,
+} from "@trussworks/react-uswds";
+
+export const SubHeader = () => {
+  const [isOpen, setIsOpen] = useState([false, false]);
+  const onToggle = (index: number): void => {
```

Comment:
Change this to `const onToggle = (index)  => {....`. What is there now is Typescript syntax.